notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -3441,6 +3441,58 @@ grief [Quinn paid tribute to his fallen comrades at the film's conclusion.]
 powered exoskeleton [The film concluded with Quinn obtaining a powerful, Predator style exoskeleton suit.]
 
 
+movie: Godzilla: The Planet Eater (2018)
+========================================
+
+:: Title
+Godzilla: The Planet Eater
+
+:: Description
+Godzilla: The Planet Eater (GODZILLA 星を喰う者, Gojira: Hoshi o Kū Mono) is a
+2018 Japanese computer-animated kaiju film directed by Kōbun Shizuno and
+Hiroyuki Seshita, written by Gen Urobuchi, and produced and animated by Toho
+Animation and Polygon Pictures, in association with Netflix. It is the 34th
+film in the Godzilla franchise, the 32nd Godzilla film produced by Toho, the
+final film in the franchise's anime trilogy, and the fourth film in the
+franchise's Reiwa period.
+
+Synopsis: A remnant of humanity, their extraterrestrial allies, and Godzilla
+battle to survive against King Ghidorah.
+
+:: Date
+2018-11-03
+
+:: References
+https://en.wikipedia.org/wiki/Godzilla:_The_Planet_Eater
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+giant monster [The titular giant dinosaur-like creature Godzilla had taken dominion over the Earth. The giant three-headed dragon-like creature Ghidorah attacked the Earth.]
+human vs. monster [A group of humans we fighting to reclaim the Earth from the giant dinosaur-like, heat ray shooting creature Godzilla and later the giant three-headed dragon-like creature Ghidorah.]
+religion as a control mechanism [Metphies used religion to dupe the men under Haruo's command into following Metphies' insane plan to bring the giant three-headed dragon-like monster Ghidorah to Earth.]
+a god and its people [Central to the plot was Metphies' fanatical devotion to the god of the Exif people, the giant three-headed dragon-like creature Ghidorah. It turned out that the Houtua worshiped the giant moth Mothra.]
+
+:: Minor Themes
+covert alien observers [Mephites revealed that his alien people had been secretly watching the people of Earth since the dawn of civilization.]
+heroism [Metphies prophesied that Haruo would be a hero to humanity.]
```

Comment:
Don't think that's how we use heroism. But you might have meant https://www.themeontology.org/theme.php?name=hero%20worship

Reply:
I elaborated on the motivation here. Sounds good?